system/Common.php

```diff
@@ -590,7 +592,7 @@ function function_usable(string $functionName): bool
      *
      * @throws FileNotFoundException
      */
-    function helper($filenames)
+    function helper($filenames): void
```

Comment:
@kenjis said to add PHPDoc comments

Reply:
If a BC break or not supported by current PHP version (like intersections and unions or complex callable signatures), this one is neither of those.

Reply:
Functions cannot be extended in PHP. So the change will not break user app code.